app/component/SummaryRow.js

```diff
@@ -217,20 +217,21 @@ const SummaryRow = (
   { data, breakpoint, intermediatePlaces, zones, ...props },
   { intl, intl: { formatMessage }, config },
 ) => {
-  const isTransitLeg = leg => leg.transitLeg || leg.rentedBike;
+  // const isTransitLeg = leg => leg.transitLeg || leg.rentedBike;
   const refTime = moment(props.refTime);
   const startTime = moment(data.startTime);
   const endTime = moment(data.endTime);
   const duration = endTime.diff(startTime);
   const slackDuration = getTotalSlackDuration(intermediatePlaces);
   const legs = [];
-  let noTransitLegs = true;
-
+  const noTransitLegs = true;
+  /*
   data.legs.forEach(leg => {
     if (isTransitLeg(leg)) {
       noTransitLegs = false;
     }
   });
+  */
 
```

Comment:
Instead of irreversibly commenting out existing code (which is functionality used by others) rather introduce a feature toggle, e.g. a const renderAlwaysNonTransitLegs which, if set to true (as in our case) skips the scanning loop.